frontend/src/types/item.d.ts

```diff
@@ -0,0 +1,15 @@
+export type Item = {
+    id: string,
+    name: string
+    slug: string
+    weight: Weight,
+    source?: string
+    image?: string
+    tags: Tag[]
+}
+
+export type Weight = {
+    value: number
+    additionalValue?: number
+    isCa: boolean
```

Comment:
isCa is also optional. 
```suggestion
    isCa?: boolean
```